### PR TITLE
added second team work

### PR DESCRIPTION
**Pull request by az-16**
Second work added. Please review it.

### CODE REVIEW

**Comment by LytvynOlena** [comment]
Don't even try Artur =)

**Comment by ArturZalutskyi** [comment]
what did you do here?

**Comment by az-16** [comment]
Guys it is really good review)

**Review by ArturZalutskyi — COMMENTED** [review]
looks good for me

File: src/test/java/com/cybertek/aizat_work_here/Hello.java [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
+
 public class Hello {
-    //2 day of team work
+    public static void main(String[] args) {
 
+        //2 day of team work
+        System.out.println("changed hello stuff");
 
+        System.out.println("Hello team, I am starting second time");
+    }
```
Comment (LytvynOlena) [inline src/test/java/com/cybertek/aizat_work_here/Hello.java:12]: I think you are ready =)

**Pull request merged**